apps_download/models/product_product.py

```diff
@@ -52,6 +53,14 @@ def create_dependency_list(self):
 
     @api.multi
     def generate_zip_file(self):
+        product1 = self.env.ref('apps_download.product_product_100')
```

Comment:
Why these hardcoded stuff?

Reply:
to make the test success, thing is that on original system we will have the real path for module form github interface modules. while for the test cases we havve to use the path from module location.
I am not sure if this is proper approch or not but I do not have any other alternative in mind. if you can think better solution then please suggest we will love to impliment it.

Reply:
ahh not really this is the function called from the method itself. that method is called when we download the module. (in case of test from Tour)

Reply:
OK, but it's still a bit weird. I don't insist more.